pyproject.toml

```diff
@@ -31,6 +31,7 @@ requires-python = ">=3.9"
 dependencies = [
     "numpy>=1.20.3",
     "matplotlib >=1.4.0",
+    "modflow-devtools",
```

Comment:
I'm not happy with this dependency, but it's how `generate_classes` currently downloads and unzips. Perhaps we should move it, or copy it.